docs/security/cve/zipbombs-and-octopus-deploy.md

```diff
@@ -0,0 +1,59 @@
+---
+title: ZipBombs and Octopus Deploy
```

Comment:
Just wondering, why is the word `ZipBomb` missing the space? I.e. not `Zip Bomb`?

It's done this way throughout this doc.

I don't really see it referred to that way online. There might be a reason, but if not, I thought I'd call it out 🙂

Reply:
It's also done that way in the [security advisory](https://advisories.octopus.com/post/2023/sa2023-01/), so being consistent on public-facing docs seems reasonable 🤷 

I've been styling it this way through the whole process, so just habit too?

Reply:
Cool. Consistency is good. Whatever you reckon 🙂